frontend/src/pages/org/crawl-detail.ts

```diff
@@ -893,7 +894,7 @@ ${this.crawl?.description}
       return;
     }
     try {
-      this.logs = await this.getCrawlLogs(params);
+      this.logs = await this.getCrawlErrors(params);
     } catch {
       this.notify({
```

Comment:
It might be good to only display this if it's not a 503? Otherwise the notification right when the crawl kicks off might be a bit concerning, even though it's expected.